library.properties

```diff
@@ -1,5 +1,5 @@
 name=SparkFun AS7331 Arduino Library
-version=2.0.0
+version=2.1.0
```

Comment:
I'd go with 2.0.1, this is more of a bug fix than a new feature or anything like that.

Reply:
Since it changes fundamentally how things work in terms of organization, it's not compatible with 2.0.X anymore. It's not enough of a change to warrant a major rev, so I think it fits.